### ISSUE TITLE
Async parameter ignored in HttpMessage constructor

### ISSUE THREAD

**Issue by ufoscout** [Jasig/cas#858]
In the org.jasig.cas.util.http.HttpMessage class in the master, the "async" parameter in the constructor is non used, so the default value is always used. Here the code:
```
public HttpMessage(final URL url, final String message, final boolean async) {
        this.url = url;
        this.message = message;
    }
```
there is a missing: this.async = async;

Cheers

**Comment by leleuj** [comment]
Good catch! Would you mind proposing a small PR to fix that?

**Comment by ufoscout** [comment]
Here your simple fix: #862.
